source/agora/consensus/Genesis.d

```diff
@@ -0,0 +1,68 @@
+/*******************************************************************************
+
+    Contains primitives related to the genesis block
+
+    Copyright:
+        Copyright (c) 2019 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+module agora.consensus.Genesis;
+
+import agora.common.Block;
+import agora.common.Data;
+import agora.common.Hash;
+import agora.common.Transaction;
+import agora.common.crypto.Key;
+
+/*******************************************************************************
+
+    Creates the genesis block.
+    The output address is currently hardcoded to a randomly generated value,
+    it will be replaced later with the proper address.
```

Comment:
Technically won't it be replaced with a set of outputs?

Reply:
I just moved it man, don't shoot the messenger